xrviz/fields.py

```diff
@@ -136,3 +148,32 @@ def set_coords(self, data, var):
 
     def ndim_matches(self, var1, var2):
         return self.data[var1].ndim == self.data[var2].ndim
+
+    def check_are_var_coords(self):
+        '''
+        Check if both x and y are in variable's coords
+        '''
+        var_coords = list(self.data[self.var].coords)
+        x = self.x.value
+        y = self.y.value
+        return True if x in var_coords and y in var_coords else False
+
+    def guess_x_y(self, var):
+        try:
+            parsed_var = self.data.metpy.parse_cf(var)
+            x, y = parsed_var.metpy.coordinates('x', 'y')
+            return [coord.name for coord in (x, y)]
+        except:  # fails when coords have not been set or available.
+            return [None, None]
+
+
+def has_metpy():
+    global mpcalc
+    try:
+        import metpy.calc as mpcalc
+    except:
+        warnings.warn("""Install Metpy using `pip install metpy` or 
+                      `conda install -c conda-forge metpy pint pooch --no-deps`
+                      to automatically guess values of x,y""",
```

Comment:
`automatically guess values of x,y` - there is no context, the user won't know why this appeared. Wording could be like
"
Metpy is not installed, so xrviz cannot automatically guess the best dimensions for output x and y axes.
To install metpy, you can execute the following:
...
"

Reply:
Changed according to above comment. Thanks!